selftests/functional/test_basic.py

```diff
@@ -609,7 +610,7 @@ class RunnerSimpleTest(unittest.TestCase):
     def setUp(self):
         self.tmpdir = tempfile.mkdtemp(prefix='avocado_' + __name__)
         self.pass_script = script.TemporaryScript(
-            'avocado_pass.sh',
+            'ʊʋʉʈɑʅʛʌ',
```

Comment:
This is a good assertion that we're handling unicode test references.  In the future we may even think about a fuzzer for this kind of input.